diff --git a/_establishment/list/menu1_list.php b/_establishment/list/menu1_list.php
index 34cf828bd1b382ca6c53ece8abe0301bc17853e5..172e905ee7625ad2f26b598016e9f2c6cb28b315 100644
--- a/_establishment/list/menu1_list.php
+++ b/_establishment/list/menu1_list.php
@@ -1,51 +1,77 @@
 <?php 
 include_once('../../_common.php');
 define('menu_establishment', true);
 include_once(NONE_PATH.'/header.php'); 
 
 if(!$_GET['date']) {
 	$date = date('Y-m');
 }
 $preDate = date('Y-m', strtotime($date." -1 Month"));
 $nxtDate = date('Y-m', strtotime($date." +1 Month"));
 
 $searchDate = date('Y-m-t', strtotime($date));
 
 $sql_common = " from  {$none['worksite']}  ";
 //22.03.10 진행중, 완료로만 표시하도록 변경 수정  nw_sdate >= '".$date."-01'
 $sql_search = " where (1) "; 
 
 //22.08.31 실행부 소속일 경우 본인이 소장으로 있는 현장만 출력
-if($member['mb_2'] == 10) {
-	$sql_search .= " and (nw_ptype1_1 = '{$member['mb_id']}' or nw_ptype1_2 = '{$member['mb_id']}' or nw_ptype1_3 = '{$member['mb_id']}' or nw_ptype2_1 = '{$member['mb_id']}' or nw_ptype2_2 = '{$member['mb_id']}' or nw_ptype2_3 = '{$member['mb_id']}') ";
-}
+if($member['mb_2'] == 10) {
+        $sql_search .= " and (
+            nw_ptype1_1 = '{$member['mb_id']}' or
+            nw_ptype1_2 = '{$member['mb_id']}' or
+            nw_ptype1_3 = '{$member['mb_id']}' or
+            nw_ptype1_4 = '{$member['mb_id']}' or
+            nw_ptype1_5 = '{$member['mb_id']}' or
+            nw_ptype1_6 = '{$member['mb_id']}' or
+            nw_ptype2_1 = '{$member['mb_id']}' or
+            nw_ptype2_2 = '{$member['mb_id']}' or
+            nw_ptype2_3 = '{$member['mb_id']}' or
+            nw_ptype2_4 = '{$member['mb_id']}' or
+            nw_ptype2_5 = '{$member['mb_id']}' or
+            nw_ptype2_6 = '{$member['mb_id']}'
+        ) ";
+}
 
 //23.07.14 현장소장 권한일경우 본인 현장 나오도록 
-if($member['mb_level2'] == 2) {
-	$sql_search .= " and (nw_ptype1_1 = '{$member['mb_id']}' or nw_ptype1_2 = '{$member['mb_id']}' or nw_ptype1_3 = '{$member['mb_id']}' or nw_ptype2_1 = '{$member['mb_id']}' or nw_ptype2_2 = '{$member['mb_id']}' or nw_ptype2_3 = '{$member['mb_id']}') ";
-}
+if($member['mb_level2'] == 2) {
+        $sql_search .= " and (
+            nw_ptype1_1 = '{$member['mb_id']}' or
+            nw_ptype1_2 = '{$member['mb_id']}' or
+            nw_ptype1_3 = '{$member['mb_id']}' or
+            nw_ptype1_4 = '{$member['mb_id']}' or
+            nw_ptype1_5 = '{$member['mb_id']}' or
+            nw_ptype1_6 = '{$member['mb_id']}' or
+            nw_ptype2_1 = '{$member['mb_id']}' or
+            nw_ptype2_2 = '{$member['mb_id']}' or
+            nw_ptype2_3 = '{$member['mb_id']}' or
+            nw_ptype2_4 = '{$member['mb_id']}' or
+            nw_ptype2_5 = '{$member['mb_id']}' or
+            nw_ptype2_6 = '{$member['mb_id']}'
+        ) ";
+}
 
 if(!$status) $status = 1;
 
 if($status == 1) {
 	$sql_search .= " and  nw_status = 0 "; 
 } else if($status == 2) {
 	$sql_search .= " and  nw_status = 1 "; 
 } else if($status == 3) {
 	$sql_search .= " "; 
 } else {
 	$sql_search .= " and  nw_status = 0 "; 
 }
 
 if($stx) {
 	$sql_search .= " and  (nw_subject LIKE '%$stx%' OR nw_code LIKE '%$stx%') "; 
 }
 
 if (!$sst) {
 	$sst  = "nw_code";
 	$sod = "desc";
 }
 $sql_order = " order by $sst $sod ";
 
 $sql = " select count(*) as cnt {$sql_common} {$sql_search} {$sql_order} ";
 $row = sql_fetch($sql);
