diff --git a/_establishment/list/menu1_list.php b/_establishment/list/menu1_list.php
index 89afbeec351107b6b225c42f37f1428f6a9bdf56..0f08df7f628eca8071e0408920c1bb86b5a07e7f 100644
--- a/_establishment/list/menu1_list.php
+++ b/_establishment/list/menu1_list.php
@@ -222,53 +222,55 @@ $qstr .= "&status=$status&date=$date";
 													$billing1 = "<span class=\"billing_class1\">미작성</span>";
 											}
 											
 											// 새로운 기성청구서 확인 상태 (confirm4)
 											switch($nomu['confirm4']) {
 												case 0:
 													$billing2 = "<span class=\"billing_class1\">미확인</span>";
 													break;
 												case 1:
 													$billing2 = "<span class=\"billing_class3\">확인</span>";
 													break;
 												default:
 													$billing2 = "<span class=\"billing_class1\">미확인</span>";
 											}
 											
 										?>
 										<tr>
 											<td class="text-center"><?php echo date('m', strtotime($date))?>월</td>
 											<td class="text-center">
 												<a href="/_worksite/write/menu1_write.php?w=u&seq=<?php echo $row['seq']?>" target="_blank">
 													<?php echo $row['nw_code']?>
 												</a>
 											</td>
 											<td><?php echo $row['nw_subject']?></td>
 											<td class="text-center"><a href="../write/menu1_write.php?w=<?php echo $w?>&seq=<?php echo $row['seq']?>&date=<?php echo $date?>&index=1">작성하기</a></td>
-											<td class="text-center">
-												<a href="./download.php?type=excel&seq=<?php echo $row['seq']?>&date=<?php echo $date?>" title="엑셀 다운로드" style="color:#444"><span class="glyphicon fa fa-file-excel-o"></span></a>&nbsp;&nbsp;
-											</td>
+                                        <td class="text-center">
+                                                <a href="./download.php?type=excel&seq=<?php echo $row['seq']?>&date=<?php echo $date?>" title="엑셀 저장" style="color:#444">
+                                                        <span class="glyphicon fa fa-file-excel-o"></span> 엑셀 저장
+                                                </a>
+                                        </td>
 											<td class="text-center">
 												<a href="./data_download.php?type=all&seq=<?php echo $row['seq']?>&date=<?php echo $date?>" title="일괄 다운로드" style="color:#444"><span class="glyphicon fa fa-download"></span></a>
 											</td>
 											<td class="text-center">
 												<?php echo $billing1 ?> | <?php echo $billing2 ?>
 											</td>
 											<td class="text-center">
 												<?php echo $noim1?> | <?php echo $noim2?>
 											</td>
 											<td class="text-center">
 												<?php echo $status?>
 											</td>
 											<td class="text-center">
         <a href="javascript:data_delete(<?php echo $row['seq']?>)" title="일괄삭제"  style="color:red"><span class="glyphicon fa fa-trash-o">🗑️</span></a>
 											</td>
 										</tr>
 										<?php } ?>
 										
 										<?php if($i == 0) {?> 
 										<tr>
 											<td colspan="11" class="align-center">검색 된 데이터가 없습니다.</td> 
 										</tr>
 										<?php }?>
 									</tbody>
 									</table>
